src/components/MyAccount/ProfileHeader.tsx

```diff
@@ -10,18 +11,31 @@ import styles from "../../../styles/components/MyAccount.module.scss"
 import type { Patron } from "../../types/myAccountTypes"
 import type { IconListElementPropType } from "./IconListElement"
 import { buildListElementsWithIcons } from "./IconListElement"
+import UsernameForm from "./NewSettings/UsernameForm"
+import { useEffect, useRef, useState } from "react"
+import type { StatusType } from "./NewSettings/StatusBanner"
+import { StatusBanner } from "./NewSettings/StatusBanner"
 
 const ProfileHeader = ({ patron }: { patron: Patron }) => {
   const { isLargerThanMobile } = useNYPLBreakpoints()
+  const [usernameStatus, setUsernameStatus] = useState<StatusType>("")
+  const [usernameStatusMessage, setUsernameStatusMessage] = useState<string>("")
+  const usernameBannerRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (usernameStatus !== "" && usernameBannerRef.current) {
+      usernameBannerRef.current.focus()
+    }
+  }, [usernameStatus])
+
+  const usernameState = {
+    setUsernameStatus,
+    setUsernameStatusMessage,
+  }
 
   const profileData = (
     [
       { icon: "actionIdentityFilled", term: "Name", description: patron.name },
-      {
```

Comment:
Did you try passing in 
`<UsernameForm patron={patron} usernameState={usernameState} />` as the description instead of escaping this pattern?

Reply:
So true

Reply:
Alignment is off now but working on that